ktor-client/ktor-client-core/common/src/io/ktor/client/features/HttpClientFeature.kt

```diff
@@ -43,6 +43,8 @@ public fun <B : Any, F : Any> HttpClient.feature(feature: HttpClientFeature<B, F
  * @throws [IllegalStateException] if [feature] is not installed.
  */
 public operator fun <B : Any, F: Any> HttpClient.get(feature: HttpClientFeature<B, F>): F {
+    feature(feature)?.let {  return it }
```

Comment:
I don't know is it idiomatic in Kotlin or not but this `.let {  return it }` hides the intent of early return.

Reply:
Agree, will change